kernel/src/actions/deletion_vector.rs

```diff
@@ -283,6 +302,22 @@ mod tests {
         assert_eq!(dv_url, example.absolute_path(&parent).unwrap().unwrap());
     }
 
+    #[test]
+    fn test_inline_read() {
+        let inline = dv_inline();
+        let sync_engine = SyncEngine::new();
+        let fs_client = sync_engine.get_file_system_client();
+        let parent = Url::parse("http://not.used").unwrap();
+        let tree_map = inline.read(fs_client, &parent).unwrap();
+        assert_eq!(tree_map.len(), 6);
+        for i in [3, 4, 7, 11, 18, 29] {
+            assert!(tree_map.contains(i));
+        }
+        for i in [1, 2, 8, 17, 55, 200] {
+            assert!(!tree_map.contains(i));
+        }
```

Comment:
Do we actually worry about a case where this assertion might fail even tho the previous two assertions (length and content) succeeded?

Reply:
I guess this is mostly just testing that the treemap is sane at this point. I don't really worry about it, but figured a few negative tests wouldn't hurt.